container_registry.container_analysis.samples_test.TestContainerAnalysisSamples: test_pubsub failed

container_registry.container_analysis.samples_test.TestContainerAnalysisSamples: test_pubsub failed
buildID: 47af18df093bd95ff650f8a10564ccee2e32b5e8
buildURL: https://source.cloud.google.com/results/invocations/27c6506f-a1f8-4829-921a-627b8fabd8ac
status: failed

re-opening due to another failure in [the build](https://source.cloud.google.com/results/invocations/98cc2144-7b0e-42ef-9506-b93b7351600a/targets).

Log:

```
Traceback (most recent call last):
  File "/tmpfs/src/github/python-docs-samples/container_registry/container_analysis/samples_test.py", line 180, in test_pubsub
    assert message_count <= receiver.msg_count
AssertionError: assert 1 <= 0
 +  where 0 = <samples_test.MessageReceiver object at 0x7fad8c026ca0>.msg_count
```

--- a/container_registry/container_analysis/samples_test.py
+++ b/container_registry/container_analysis/samples_test.py
@@ -29,6 +29,7 @@ from grafeas.grafeas_v1.gapic.enums import DiscoveryOccurrence
 from grafeas.grafeas_v1.gapic.enums import NoteKind
 from grafeas.grafeas_v1.gapic.enums import Severity
 from grafeas.grafeas_v1.gapic.enums import Version
+import pytest
 
 import samples
 
@@ -145,6 +146,7 @@ class TestContainerAnalysisSamples:
         # clean up
         samples.delete_occurrence(basename(occ.name), PROJECT_ID)
 
+    @pytest.mark.flaky(max_runs=3, min_passes=1)
     def test_pubsub(self):
         # create topic if needed
         client = SubscriberClient()

